src/main/java/Storage.java

```diff
@@ -0,0 +1,94 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Storage {
+    private final String filePath;
+    private File file;
+
+    public Storage(String filePath) {
+        this.filePath = filePath;
+        this.file = new File(filePath);
+    }
+
+    /** Helper: creates file if it is not present in filePath.
+     * @throws IOException
+     */

```

Comment:
Here the Javadocs opening /** should be on a separate line and there should be a blank line between the description and parameter section